src/checkpoint_test.rs

```diff
@@ -0,0 +1,108 @@
+use crate::Checkpoint;
+use serial_test::serial;
+
+#[test]
+#[serial]
+fn create() {
+    crate::remove("VAR");
```

Comment:
if you look at other tests i wrote in this crate, each test uses different env vars. the env var name prefixed by the test name.
this allows me to run all tests in parallel which is much faster.

Reply:
Yes that's true, but due to the very nature of the checkpoint system we could see unintended side-effects (I did while testing, which is why I forced serial).

```
Test 1 (T1): env::set("ENV1", "1")
T1: checkpoint()

Test 2 (T2): env::set("ENV2", "2")
T2: checkpoint()
T2: env::set("ENV2", "3")

T2: restore()
T1: restore()

T2: assert!("ENV2" == "2")
^ This will fail, because T1 restores "ENV2", which will remove the value
```

Reply:
ah ok. so use the cargo-make built in capability